### PR TITLE
Improved UI of motion list and detail view

### PR DESCRIPTION
**Pull request by emanuelschuetze**

### CODE REVIEW

File: client/src/app/site/motions/services/local-permissions.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { TestBed } from '@angular/core/testing';
+
+import { LocalPermissionsService } from './local-permissions.service';
```
Comment (tsiegleauq) [inline client/src/app/site/motions/services/local-permissions.service.spec.ts]: e2e imports missing. Tests will fail without

File: client/src/app/site/motions/services/motion-repository.service.ts [hunk 2]
```diff
@@ -152,11 +152,23 @@ export class MotionRepositoryService extends BaseRepository<ViewMotion, Motion>
...
      */
-    public async setRecommenderState(viewMotion: ViewMotion, stateId: number): Promise<void> {
+    public async setRecommendation(viewMotion: ViewMotion, recommendationId: number): Promise<void> {
         const restPath = `/rest/motions/motion/${viewMotion.id}/set_recommendation/`;
-        await this.httpService.put(restPath, { recommendation: stateId });
+        await this.httpService.put(restPath, { recommendation: recommendationId });
+    }
+
+    /**
+     * Set the state of a motion
```
Comment (tsiegleauq) [inline client/src/app/site/motions/services/motion-repository.service.ts]: wrong comment.

File: client/src/app/site/motions/services/motion-repository.service.ts [hunk 3]
```diff
@@ -152,11 +152,23 @@ export class MotionRepositoryService extends BaseRepository<ViewMotion, Motion>
...
+     *
+     * @param viewMotion target motion
+     * @param stateId the number that indicates the state
+     */
+    public async setCatetory(viewMotion: ViewMotion, categoryId: number): Promise<void> {
+        const motion = viewMotion.motion;
+        motion.category_id = categoryId;
+        await this.update(motion, viewMotion);
```
Comment (tsiegleauq) [inline client/src/app/site/motions/services/motion-repository.service.ts]: of this function should be executed on multiple motions, that "might" need a
`return await` so synchronize the execution of the loops

File: client/src/styles.scss [hunk 4]
```diff
@@ -188,3 +195,47 @@ mat-panel-title mat-icon {
...
+    margin: 8px 8px 8px 0;
+}
+.mat-chip:focus, .mat-basic-chip:focus {
+    outline: none;
+}
+button.mat-menu-item.selected {
+    font-weight: bold !important;
+}
+
+/** Colors **/
```
Comment (tsiegleauq) [inline client/src/styles.scss]: Prettier can align the CSS files for you.

in doubt (missing empty line):
```
    ...
    color: white !important;
}

.darkblue { ...
```

**Comment by tsiegleauq** [comment]
cluttered side:
localhost_4200_history 1

File: client/src/app/site/motions/components/motion-detail/motion-detail.component.ts [hunk 5]
```diff
@@ -609,14 +609,18 @@ export class MotionDetailComponent extends BaseViewComponent implements OnInit {
      * Supports the motion (as requested user)
      */
     public support(): void {
-        this.repo.support(this.motion);
+        this.repo.support(this.motion).then((response: any) => {
```
Comment (FinnStutzenstein) [inline client/src/app/site/motions/components/motion-detail/motion-detail.component.ts]: `.then(null, this.raiseError)`.

File: client/src/app/site/motions/components/motion-detail/motion-detail.component.ts [hunk 6]
```diff
@@ -609,14 +609,18 @@ export class MotionDetailComponent extends BaseViewComponent implements OnInit {
...
+            this.raiseError(response.detail);
+        });
     }
 
     /**
      * Unsupports the motion
      */
     public unsupport(): void {
-        this.repo.unsupport(this.motion);
+        this.repo.unsupport(this.motion).then((response: any) => {
```
Comment (FinnStutzenstein) [inline client/src/app/site/motions/components/motion-detail/motion-detail.component.ts]: See above.

**Pull request merged**